gcp/modules/couchdb/main.tf

```diff
@@ -60,22 +60,23 @@ resource "null_resource" "couchdb_finish_cluster" {
 
   provisioner "local-exec" {
     command = <<EOF
-      RETRIES=10
+      RETRIES=15
       RETRY_COUNT=1
-      while [ "$PODS_READY" != "true" ]; do
-        PODS_READY="true"
-        echo "[Try $RETRY_COUNT of $RETRIES] Waiting for all CouchDB pods to become Running..."
-        for STATUS in $(kubectl get pods --namespace ${var.release_namespace} -l app=couchdb -o jsonpath='{.items[*].status.phase}'); do
-          if [ "$STATUS" != "Running" ]; then
-            PODS_READY="false"
+      while [ "$CLUSTER_READY" != "true" ]; do
+        CLUSTER_READY="true"
+        echo "[Try $RETRY_COUNT of $RETRIES] Waiting for all CouchDB pods to join the cluster..."
+        for CLUSTER_MEMBERS in $(kubectl exec --namespace gpii couchdb-couchdb-0 -c couchdb -- curl -s http://${var.secret_couchdb_admin_username}:${var.secret_couchdb_admin_password}@127.0.0.1:5984/_membership 2>/dev/null | jq -r .cluster_nodes[] | grep -c .); do
```

Comment:
what happens if the couchdb-couchdb-0 is not ready (gives a "connection timed out" or a wrong output that it is not in JSON format), does this part of the script fail?

Reply:
In that case script will just sleep and retry again:
* We redirect all error output to `/dev/null`
* `jq` returns nothing to stdout in case of incorrect input
* `grep -c` returns 0 if there is nothing in stdin